.github/workflows/packaging.yml

```
@@ -0,0 +1,36 @@
+name: Package
+# This workflow is triggered on pushes to the repository.
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  package:
+    name: Package and push
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
+      - name: Install bumpversion dependency
+        run: |
+          python -m pip install --upgrade pip
+          pip install bump2version pyyaml
```

This line could be something like:
```suggestion
          pip install bump2version pyyaml cloudve-packaging-utils
```

---

`pip` feels a bit weird for it given that there are `sh` scripts, but what if we just had a separate git repository with the scripts externalized to it and have this CI `git clone` that repo and run them?